### ISSUE TITLE
Advance Geo-Search

### ISSUE THREAD

**Issue by iamraphson** [algolia/vue-instantsearch#370]
can I do a component like https://preview.algolia.com/geo-search/ for vue. The more, you draw or drag, the more it changes the result

**Comment by Haroenv** [comment]
It's not part of the components we have right now, but you can make a custom component which changes the relevant refinements. See more https://community.algolia.com/vue-instantsearch/getting-started/search-store.html

cc @samouss who is making that kind of functionality for React InstantSearch

**Comment by samouss** [comment]
Yep we plan to implement this kind of widget in each InstantSearch library. We keep this issue as a reminder, we will link the next discussions about GeoSearch to it.

**Comment by iamraphson** [comment]
@samouss i will be looking forward to the widget.

**Comment by michtio** [comment]
any updates on this one , like an ETA ?

**Comment by Haroenv** [comment]
sooner rather than later, but no strict ETA, hope you understand @michtio

**Comment by Haroenv** [comment]
The widget itself won't be built-in to the initial release of v2, but you will be able to use the connector and make your own widget using third-party widgets. 

[![Edit vue-instantsearch v2 alpha](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/3vqr1k0831?module=%2Fsrc%2FMyGeoSearch.vue)

Really soon (after a few more naming changes) we will release the beta version for v2, alpha is already released. After that we will focus on server side rendering so that we can have a main field release after that

**Comment by ul8** [comment]
Thanks for the update @Haroenv !
